Packs/TrustwaveFusion/Integrations/TrustwaveFusion/TrustwaveFusion.yml

```diff
@@ -1,4 +1,4 @@
-category: Data Enrichment & Threat Intelligence
+category: Email
```

Comment:
The Trustwave Fusion Integration is not an email integration.
TrustwaveSEG is Trustwave's Secure Email Gateway product.

Reply:
Trustwave Fusion category will remain Data Enrichment & Threat Intelligence, it was removed from this pr.